cmd/registry/controller/local-tasks.go

```diff
@@ -26,16 +38,57 @@ type ExecCommandTask struct {
 	TaskID string
 }
 
+func getCommand(ctx context.Context, action []string) (*cobra.Command, error) {
+	if len(action) == 0 {
+		return nil, fmt.Errorf("Empty action string")
+	}
+
+	switch action[0] {
+	case "annotate":
+		return annotate.Command(ctx), nil
+	case "compute":
+		return compute.Command(ctx), nil
+	case "delete":
+		return delete.Command(ctx), nil
+	case "export":
+		return export.Command(ctx), nil
+	case "get":
+		return get.Command(ctx), nil
+	case "index":
+		return index.Command(ctx), nil
+	case "label":
+		return label.Command(ctx), nil
+	case "list":
+		return list.Command(ctx), nil
+	case "search":
+		return search.Command(ctx), nil
+	case "upload":
+		return upload.Command(ctx), nil
+	case "vocabulary":
+		return vocabulary.Command(ctx), nil
```

Comment:
If we ever change these subcommand names, will there be a test that points out this connection is broken?

Reply:
Added another bug to take care to the tests https://github.com/apigee/registry/issues/248